homeassistant/helpers/intent.py

```diff
@@ -81,13 +89,29 @@ class IntentHandleError(IntentError):
     pass
 
 
+@callback
+@bind_hass
+def _match_entity(hass, name):
+    """Match a name to an entity."""
+    from fuzzywuzzy import process as fuzzyExtract
```

Comment:
This is installed as part of the conversation component. You can't use it here.

The solution is that now we're going to need to add fuzzywuzzy as a core dependency. I just don't know if I like that.

Is the added value of fuzzywuzzy that much that we need it? Could we just map against name?

Reply:
Well, I put it in conversation so it would get added somewhere since I can't add a requirement to a helper.

We could. Would be a little more convoluted with some regexs but I would guess we can cover most cases fairly easily.

Reply:
If the user doesn't have the conversation component enabled, it will not install the dependency.